internal/graph/checkingresourcestream.go

```diff
@@ -466,7 +471,15 @@ func (crs *checkingResourceStream) runProcess(alwaysProcess bool) (bool, error)
 		// Set the lookupResult iff the permissionship was a valid permission.
 		var lookupResult *v1.DispatchLookupResourcesResponse
 		if permissionship != v1.ResolvedResource_UNKNOWN {
-			metadata := combineResponseMetadata(rai.reachableResult.Metadata, resultsMetas[rai.reachableResult.Resource.ResourceId])
+			// NOTE: since the metadata returned for the check will encompass the dispatches shared for all results, only
```

Comment:
The assumption here is that the bulk check will return the dispatches on every result object? Could we instead return a metadata object that applies to the entire result set?

Reply:
Not if debugging is enabled, no

Reply:
I changed to returning a single metadata, but only if debugging is disabled. Otherwise, chunking is disabled.